Demo/Fullscreen/Review/ReviewViewDemoView.swift

```diff
@@ -31,7 +31,13 @@ public class ReviewViewDemoView: UIView {
 
 extension ReviewViewDemoView: ReviewViewDelegate {
     public func reviewView(_ reviewView: ReviewView, didSelect profile: ReviewViewProfileModel) {
+        LoadingView.show()
+        reviewView.setSelectButtonEnabled(false)
         print("Did select: \(profile.name) for review")
```

Comment:
Debug code?

Reply:
Demo code 😄 Did not change this line, though

Reply:
Yes probably leftover